test/functional/test_framework/test_node.py

```diff
@@ -74,8 +75,11 @@ def __init__(self, i, dirname, extra_args, rpchost, timewait, binary, stderr, mo
 
     def __getattr__(self, name):
         """Dispatches any unrecognised messages to the RPC connection."""
```

Comment:
In commit "[tests] allow tests to be run with --usecli"

Improve comment.

Reply:
fixed